src/Sentry/SentryClient.cs

```diff
@@ -129,6 +129,17 @@ public void CaptureTransaction(ITransaction transaction)
                 return;
             }
 
+            if (!transaction.IsFinished)
+            {
+                _options.DiagnosticLogger?.LogWarning(
```

Comment:
Loggin a warning makes sense.

Should we just finish the transaction and capture it instead?

How does it work in the other SDKs? Could be a behavior we align.

Reply:
that's a good point, we don't check it at all on Java

Reply:
what's about logging it as `INFO` instead but finishing and capturing it?

Reply:
Ive raised this issue https://github.com/getsentry/sentry-java/issues/1208

Reply:
Changed it so that it captures with a warning.